docs/literate/src/files/subcell_shock_capturing.jl

```diff
@@ -0,0 +1,104 @@
+#src # Subcell limiting with the IDP Limiter
+
+# In the previous tutorial, the element-wise limiting with [`IndicatorHennemannGassner`](@ref)
+# and [`VolumeIntegralShockCapturingHG`](@ref) was explained. This tutorial contains a short
+# introduction to the idea and implementation of subcell shock capturing approaches in Trixi.jl,
+# which is also based on the DGSEM scheme in flux differencing formulation.
+# Trixi.jl contains the a-posteriori invariant domain-preserving (IDP) limiter which was
+# introduced by [Rueda-Ramírez, Pazner, Gassner (2022)](https://doi.org/10.1016/j.compfluid.2022.105627)
+# and [Pazner (2020)](https://doi.org/10.1016/j.cma.2021.113876). It is a flux-corrected
+# transport-type (FCT) limiter and is implemented using [`SubcellLimiterIDP`](@ref) and
+# [`VolumeIntegralSubcellLimiting`](@ref).
+# Since it is an a-posteriori limiter you have to apply a correction stage after each Runge-Kutta
+# stage. This is done by passing the stage callback [`SubcellLimiterIDPCorrection`](@ref) to the
+# time integration method.
+
+# ## Time integration method
+# As mentioned before, the IDP limiting is an a-posteriori limiter. Its limiting process
+# guarantees the target bounds for a simple Euler evolution. To still achieve a high-order
+# approximation, the implementation uses strong-stability preserving (SSP) Runge-Kutta method
+# which can be written as convex combination of these forward Euler steps.
+#-
+# Due to this functionality of the limiting procedure the correcting stage and therefore the stage
+# callbacks has to be applied to the solution after the forward Euler step and before further
+# computation. Unfortunately, the `solve(...)` routines of
+# [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl), which is normally used in
+# Trixi.jl for the time integration, does not support calculations via callback at this point
+# in the simulation.
+#-
+# Therefore, subcell limiting with the IDP limiter requires the use of a Trixi-intern
+# time integration SSPRK method called with
+# ````julia
+# Trixi.solve(ode, method(stage_callbacks = stage_callbacks); ...)`.
+# ````
+#-
+# Right now, only the third-order SSPRK method [`Trixi.SimpleSSPRK33`](@ref) is implemented.
```

Comment:
```suggestion
# Right now, only the canonical three-stage, third-order SSPRK method (Shu-Osher)  [`Trixi.SimpleSSPRK33`](@ref) is implemented.
```

Reply:
Done in [d5259c5](https://github.com/trixi-framework/Trixi.jl/pull/1882/commits/d5259c5eb584d570d13ce77f9a34682b8ad2edae). Thanks